TombLib/TombLib/Catalogs/TEN Node Catalogs/Timers.lua

```diff
@@ -0,0 +1,274 @@
+local Timer = require("Engine.Timer")
+LevelFuncs.noA = function() end
+
+-- !Name "Create Timer with Function"
```

Comment:
Function and Timer words should not be capitalized. Stick to this rule onwards please.